gcp/table_gcp_service_account_key.go

```diff
@@ -161,7 +162,9 @@ func getGcpServiceAccountKey(ctx context.Context, d *plugin.QueryData, h *plugin
 
 	keyName := "projects/" + project + "/serviceAccounts/" + serviceAccountName + "/keys/" + name
 
-	op, err := service.Projects.ServiceAccounts.Keys.Get(keyName).Do()
+	queryParameter := googleapi.QueryParameter("publicKeyType", "TYPE_RAW_PUBLIC_KEY", "TYPE_X509_PEM_FILE")
```

Comment:
Can we add any doc/api references as comment or something? Why are we passing these parameters? 

![image](https://github.com/turbot/steampipe-plugin-gcp/assets/26873346/1bfb94f6-5c66-42f9-be0a-a8434006cf66)